Make timesyncd a "simple" service

the daemon ends up waiting 100 msec on networking conditions before notifying
that it is done, even though that's really not relevant


--- systemd-233/units/systemd-timesyncd.service.in~	2017-03-01 21:43:06.000000000 +0000
+++ systemd-233/units/systemd-timesyncd.service.in	2017-05-21 13:58:02.419171076 +0000
@@ -18,7 +18,7 @@
 Wants=time-sync.target
 
 [Service]
-Type=notify
+Type=simple
 Restart=always
 RestartSec=0
 ExecStart=@rootlibexecdir@/systemd-timesyncd
